### ISSUE TITLE
Internal server error when cancelling a carpool

### ISSUE THREAD

**Issue by jillh510** [iandees/carpools#22]
I cancelled my carpool, filled in the reason, and clicked "cancel". Result: "Internal Server Error".

![screen shot 2017-04-16 at 8 07 42 am](https://cloud.githubusercontent.com/assets/2319837/25072098/c360a3fc-227b-11e7-9af2-b346b4ce3b03.png)

**Comment by jillh510** [comment]
I'm still seeing this bug.

**Comment by iandees** [comment]
```
2017-04-17T21:45:15.698725+00:00 app[web.1]: AttributeError: 'RideRequest' object has no attribute 'email'
2017-04-17T21:45:15.698724+00:00 app[web.1]:   File "/app/app/carpool/views.py", line 327, in _email_carpool_cancelled
```

**Comment by iandees** [comment]
This should be fixed in 9fad6fe.
